transcription/transcription_test.go

```diff
@@ -53,3 +55,36 @@ func TestSendEmailReturnsError(t *testing.T) {
 	err := SendEmail(username, password, host, port, to, subject, body)
 	assert.Error(err)
 }
+
+func TestDownloadFileFromURL(t *testing.T) {
```

Comment:
@yoninachmany made an issue https://github.com/qur/withmock/issues/57

Let's comment out the test for now and remove the `// mock` from the http import. Please make a github issue for us to go back and fix this test. Feel free to merge once everything else is working.